### DISCUSSION TITLE
Great Docs - Frustrating UX...

### DISCUSSION THREAD

**Discussion by corepay** [zitadel/zitadel#7791] [Ideas]
Just my personal opinion here I think your docs content is the best I've seen. -almost every feature is well described, questions are answered and scenarios help me grasp concepts tremendously.  

But....(here goes...)

Please change in-doc links to open new browser tabs.  You have many links in a page to elaborate, define and explain certain features and concepts and when clicking through in the same tab I'll find myself on another well documented page with more links.  I start out trying to reason organizations and 8 pages later find myself looking at API docs for SAML compliments asking where the hell I am, how did I get here and what was I doing before falling into this rabbit hole....searching my history to figure out where i started.  

I can see the menu items and the previos/next pages opening same tab...but...

<a href="https://imgflip.com/i/8n68td"><img src="https://i.imgflip.com/8n68td.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div>

**Comment by mffap** [comment]
Appreciate the feedback. We'll have a look at it.
Am I correct with the assumption that a jump where the sidebar-nav is changing is the hardest du grasp? I would see value in opening tabs that jump to a different "main section" such as Documentation, API, Self-hosting. 
As for internal links within the sections, I would be curious to hear a couple of other opinions from users.

**Comment by corepay** [comment]
For me, personally, it is the page content links where I need to focus and dig deep.  The links in the page are not the problem, they expand, explain and relate to the MAIN topic I am engaging.  However, the linked pages have links to relevant detail, which have links, which have links.  I should probably stop on the first link and click back to get back to the topic at hand but as a noob trying to get my head around this I run into a link from a link I am not familiar with the term or topic so I click in and like I said above somehow I end up at Burger King with only vague memroes of where I started.  It would be easier - for me at least, to click a link, go to the tab scan the first few paragraphs and decide if I want to continuem close the tab and pop back into my screen or keep it open so I am reminded down the line I need to check that page out.  I thikn it's good practice if on page content to open any external links in new tab anyway - maybe just a preference but makes sense to me.

The menus on left and page navigation IMO could stay in same tab.

Nonetheless you got some real good content - I'm past the point of head scratching and now at the navel gazing and chin stroking stage trying to figure out my multi-tenant strategy.  I have a payment switch with PL tenants who have dozens to thousands of merchants (who have hundres to thousands of customers), handful of resellers/agents and maybe a couple developers.  Oh and a b2b marketplace I need to tackle too.  Off topic here so where can I go with a question on a couple tenant strategies to get advice?  Discord or here another question maybe?

**Reply by mffap** [reply]
We probably won't change that every link opens in a new tab. I get your point, though. Maybe there's an add-on (or if not here's an idea) for a browser that can be limited to a specific url.

> The menus on left and page navigation IMO could stay in same tab.

I meant more, that we could open new tabs when you go to a different section, where also the sidebar ist different (API, self-hosting, documentation). Because I get that a reader would loose the context.

**Reply by mffap** [reply]
>  Off topic here so where can I go with a question on a couple tenant strategies to get advice? Discord or here another question maybe?

Discord or here as a new discussion. Both works.

**Reply by corepay** [reply]
Cool and thanks - this was just a view from the trenches here so hope no offense, none intended.

I'll ask about the tenant strategy here in a few.  Thanks you for your help and likely a solution I've been trying to roll on my own for months now....

**Reply by fforootd** [reply]
No offense taken 😁 our docs/product can only be as good as we listen to the community 🚀 

Actually I liked you memes 😸

**Reply by fforootd** [reply]
Maybe this expresses your feelings in a meme as well 😁 

![image](https://github.com/zitadel/zitadel/assets/9879976/8bf82608-9fe4-4348-a25c-64b95304c4e2)

**Reply by corepay** [reply]
HAHA!  Good meme seen it everywhere but never directed at me!

Actually did laugh on this one.

**Reply by fforootd** [reply]
> HAHA! Good meme seen it everywhere but never directed at me!
> 
> Actually did laugh on this one.

Wonderful that I made you laugh 🥳
